src/ListView/ListView.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/**
+ * ListView Component for Patternfly React
+ */
+const ListView = ({ children, listViewClass }) => (
+  <div className={listViewClass}>
```

Comment:
+1 will update it... can we not just use `className={classNames('classA', 'classB', var1, var2')}` etc..